kinesumer.go

```diff
@@ -34,6 +34,7 @@ type Options struct {
 	MaxShardWorkers     int
 	ErrHandler          func(k.Error)
 	DefaultIteratorType string
+	StartWorkerTryTime  time.Duration
```

Comment:
Call this `ShardAcquisitionTimeout`? It's worth adding a comment for this and `PollTime`.